src/Native/Bootstrap/main.cpp

```diff
@@ -284,11 +281,10 @@ int main(int argc, char* argv[])
     if (!RtuDllMain(NULL, DLL_PROCESS_ATTACH, NULL))
         return -1;
 
-#if defined(CPPCODEGEN) || defined(PLATFORM_UNIX)
+#if defined(CPPCODEGEN)
```

Comment:
Nit: I would keep `RhpEnableConservativeStackReporting` undef its own `#ifdef / #endif` so it is a one line change to turn on RhpEnableConservativeStackReporting for ad-hoc experiments.